build.xml

```diff
@@ -347,8 +347,19 @@
         <copy todir="${dist}/openfire/jitsiVideobridge/lib">
             <fileset dir="lib">
                 <exclude name="native/**"/>
-                <exclude name="bouncycastle.jar"/>
+                <exclude name="bc*.jar"/>

```

Comment:
I think this will also exclude bccontrib-....jar which is needed when running jitsi videobridge.